packages/venia-ui/lib/components/FilterModal/CurrentFilters/currentFilters.js

```diff
@@ -33,7 +33,11 @@ const CurrentFilters = props => {
         return elements;
     }, [classes.item, filterState, removeItem]);
 
-    return <ul className={classes.root}>{filterElements}</ul>;
+    return (
+        <ul className={classes.root} aria-label="Current Filters">
```

Comment:
I believe we would want to open this for translation; please use `formatMessage` for this label. Similar to this in [AddressBookPage](https://github.com/magento/pwa-studio/blob/1494c40366538ead7cbb41c873666352986198ba/packages/venia-ui/lib/components/AddressBookPage/addressBookPage.js#L26:L29).